File: Day7/ques1.cpp
//Question Link: https://leetcode.com/problems/subtract-the-product-and-sum-of-digits-of-an-integer/

class Solution {
public:
    int subtractProductAndSum(int n) {
        int digits, sum=0,prod=1;
        while(n>0){
            digits=n%10;
            sum+=digits;
            prod*=digits;
            n=n/10;
        }
        return prod-sum;
    }
};

File: Day 21/Ques1.py
#Question Link: https://leetcode.com/problems/sign-of-the-product-of-an-array/

def signFunc(x):
        if x>0:
            return 1
        elif x<0:
            return -1 
        else:
            return 0
class Solution:
    
    def arraySign(self, nums: List[int]) -> int:
        prod=1
        for i in nums:
            prod*=i
        return (signFunc(prod))
